### DISCUSSION TITLE
Can you share me an example of performing mTLS with Curl Client?

### DISCUSSION THREAD

**Discussion by ganeshkamath89** [curl/curl#12276] [Q&A]
I have an application that uses libCurl for Digest Authentication.
For HTTPS authentication, the server has recently enabled mTLS. I need help to set the correct Curl SetOPT headers to support mTLS

**Comment by dfandrich** [comment]
If by mTLS you mean Mutual TLS, you can do that simply by adding the `--cert` option. curl automatically authenticates the server (when using a TLS protocol), but you might need to add the `--ca-cert` option depending on the signatures on the certificate the server is using. Adding `--cert` sends a client certificate to the server that it can use to authenticate the client.

**Reply by ganeshkamath89** [reply]
Thanks @dfandrich. Give me some time to try this and verify it in my solution.
